packages/feedback/src/lib/FeedbackForm.tsx

```diff
@@ -0,0 +1,159 @@
+import React, { useEffect } from 'react';
+import { TextField } from '@availity/mui-textfield';
+import { LoadingButton, Button } from '@availity/mui-button';
+import { ToggleButtonGroup, ToggleButton } from '@availity/mui-toggle-button';
+import { Grid, SvgIconProps, ToggleButtonProps, styled } from '@mui/material';
+import { FaceFrownIcon, FaceNeutralIcon, FaceSmileIcon } from '@availity/mui-icon';
+import { avRegionsApi } from '@availity/api-axios';
+import { useForm, SubmitHandler, Controller } from 'react-hook-form';
+
+interface Inputs {
+  feedback: string;
+  smileField: string;
+}
+
+interface SmileButtonProps extends ToggleButtonProps {
+  disabled: boolean;
+  Icon: (props: SvgIconProps) => JSX.Element;
+  label: string;
+  value: string;
+}
+
+interface FeedbackFormProps {
+  // eslint-disable-next-line @typescript-eslint/no-explicit-any
+  analytics: { info: (entries: any) => any };
+  appName: string;
+  handleClose: () => void;
+  loading: boolean;
+  sent: boolean;
+  setLoading: React.Dispatch<React.SetStateAction<boolean>>;
+  setSent: React.Dispatch<React.SetStateAction<boolean>>;
+}
+
+const SmileButtons = styled(ToggleButtonGroup, { name: 'AvFeedbackContainer', slot: 'SmileButtons' })({});
+
+const SmileButton = ({ disabled, Icon, label, value, ...props }: SmileButtonProps) => (
+  <div>
+    <ToggleButton title={label} value={value} {...props} disabled={disabled}>
```

Comment:
it doesn't look like title is making it's way down to the icon. We could put it directly on the icon and remove the aria-hidden, but it's probably better to go with an aria-label so it doesn't trigger the browser's built in tooltip

Reply:
Also, even if there is no tooltip I think we should instead have the label say the value. Seems confusing for AT users to have two interactive elements with a label of "What would you improve"?